setup.cfg

```diff
@@ -7,15 +7,15 @@ long_description_content_type = text/x-rst
 url = https://github.com/PyCQA/pylint
 author = Python Code Quality Authority
 author_email = code-quality@python.org
-license = GPL-2.0-or-later
+license = GPL-2.0
```

Comment:
I believe `GPL-2.0-or-later` was correct . The license explicitly states
```
... either version 2 of the License, or (at your option) any later version. ...
```

Reply:
https://github.com/PyCQA/pylint/commit/9d959abd39cd787fa7b5d68327d19d3972fe888e